### ISSUE TITLE
[Claims Details V2] Make link look like a link

### ISSUE THREAD

**Issue by davidakennedy** [department-of-veterans-affairs/va.gov-team#78897]
## Value Statement
**_As a_** user type
**_I want to_** job to be done
**_So that_** desired outcome(s)

---
### Bug

**Issue description**

URL: `/track-claims/your-claims/600351984/files`

The link on the Claims Files page that goes to the page where Veterans can ask for the VA to make a claim decision is a link that looks like a button. This isn't a WCAG violation, but this can cause problems because some assistive technology users may not be able to easily activate it. For more, [check out links in the design system](https://design.va.gov/components/link/#accessibility-considerations).

<img width="703" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWI1QXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--830db3a1f4a436e47a01272af23bf56d634f0f57/image.png" alt="View details button on Claim Files page with an arrow pointing to the button." />

Code:

```html
<a aria-label="View details about asking VA for a claim decision" title="View details about asking VA for a claim decision" class="usa-button usa-button-secondary view-details-button" href="/track-claims/your-claims/600351984/ask-va-to-decide">View Details</a>
```

[This is an a11y-defect-3](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

**Remediation guidance**

Make the link look like a regular link, of if visual distinction is needed, use an action link.

**Does this impact visual design?**

Yes.

**Relevant standards**

N/A For more, [check out links in the design system](https://design.va.gov/components/link/#accessibility-considerations).

## Definition of Done
- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Design performs design QA and verifies the implementation matches the design spec
- [ ] Accessibility specialist performs accessibility review (in code or design)
- [ ] Engineering identifies staging users required to test and shares account and credentials with design and product
- [ ] Product performs functional QA and verifies acceptance criteria was met

## Acceptance Criteria
- [ ] The link resembles a link or action link.

**Comment by skylerschain** [comment]
Added this to [this epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57104), which deals with this banner and the subsequent page.

**Comment by pmclaren19** [comment]
Reached out to the team about this ticket and @skylerschain said that we shouldnt work it yet, since the design for [this](https://github.com/department-of-veterans-affairs/va.gov-team/issues/81422) ticket would make it obsolete. Marking this ticket as wont do for now!
